<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XWindowsClass</class>
 <widget class="QMainWindow" name="XWindowsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XWindows</string>
  </property>
  <property name="windowIcon">
   <iconset resource="XWindows.qrc">
    <normaloff>:/XWindows/controller2.ico</normaloff>
    <normalon>:/XWindows/controller2.ico</normalon>
    <disabledoff>:/XWindows/controller2.ico</disabledoff>
    <disabledon>:/XWindows/controller2.ico</disabledon>
    <activeoff>:/XWindows/controller2.ico</activeoff>
    <activeon>:/XWindows/controller2.ico</activeon>
    <selectedoff>:/XWindows/controller2.ico</selectedoff>
    <selectedon>:/XWindows/controller2.ico</selectedon>:/XWindows/controller2.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:  rgb(13, 115, 130);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>391</width>
      <height>511</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="mainLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item alignment="Qt::AlignTop">
      <widget class="QLabel" name="label_logo">
       <property name="font">
        <font>
         <family>Leelawadee UI</family>
         <pointsize>45</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(244, 244, 244);
border-top: 2px solid rgb(255, 223, 32);</string>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string>XWindows</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <property name="indent">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_controller">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="XWindows.qrc">:/XWindows/ControllerON.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="margin">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="toggleBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>50</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Leelawadee</family>
         <pointsize>15</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(243, 243, 243);
width: 175px;
margin-top: 5px;</string>
       </property>
       <property name="text">
        <string>Turn OFF</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_reconnect">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 223, 32);</string>
       </property>
       <property name="text">
        <string>Attempting to reconnect...</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_savingChanges">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 223, 32);</string>
       </property>
       <property name="text">
        <string>Saving changes...</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="bottomSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="menuLayout">
     <item>
      <widget class="QPushButton" name="xboxBtn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>33</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>49</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-style: none;
margin-left: 3px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="XWindows.qrc">
         <normaloff>:/XWindows/xbox-logo.png</normaloff>:/XWindows/xbox-logo.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>45</width>
         <height>45</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="settingsBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>67</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">border-style: none;
border-right: 2px outset rgb(255, 223, 35);
border-left: 2px outset rgb(255, 223, 35);
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="XWindows.qrc">
         <normaloff>:/XWindows/settings1.png</normaloff>:/XWindows/settings1.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>45</height>
        </size>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_verNum">
       <property name="font">
        <font>
         <family>Sylfaen</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">margin-right:7px;
color: rgb(243,243,243);
border-left: 2px solid rgb(255, 223, 32);</string>
       </property>
       <property name="text">
        <string>X.X</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>560</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="footerLayout">
     <item>
      <widget class="QLabel" name="label_footer">
       <property name="font">
        <font>
         <family>Leelawadee UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 211, 32);
border-top: 2px solid rgb(255, 211, 32);</string>
       </property>
       <property name="text">
        <string>www.github.com/Henduww</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="XWindows.qrc"/>
 </resources>
 <connections/>
</ui>
